### ISSUE TITLE
Multiline info lines are represented as a single line in the minutes.

### ISSUE THREAD

**Issue by abompard** [fedora-infra/mote#687]
In [this example](https://meetbot.fedoraproject.org/meeting-3_matrix_fedoraproject-org/2024-01-15/cpe-infra-releng-daily-standup.2024-01-15-08.30.html), the first Info line is represented as one line, but if you look at [the plain text minutes](https://meetbot-raw.fedoraproject.org//meeting-3_matrix_fedoraproject-org/2024-01-15/cpe-infra-releng-daily-standup.2024-01-15-08.30.txt) you can see that it was actually 3 lines.

**Comment by sopyb** [comment]
I will be working on this issue. I have contacted t0xic0der to ask some questions about the project since I have been stuck trying to understand the code base in some places.

**Comment by gridhead** [comment]
@sopyb go ahead

**Comment by sopyb** [comment]
I've been looking through the codebase and haven't found anything related to generating HTML files for meeting message logs or summaries. There seems to be only the option to clear cache for specific paths through fedora messaging for when new data is added. The archive of meetings is no longer available to download online so I can't experiment and make sure that I am not missing something.


@gridhead My questions are:
1. Can I still get the archive from anywhere?
2. Is the code generating the summary and message logs html public (I might have missed it. Let me know in what files should I look). If so in what repository?


The best fix I can think of is replacing `\n` with`<br>` on the html generation step for content that would go into `<li>` elements. Another option for fixing this issue is, kind of a band-aid solution where I would add the style `white-space: pre-line;` to items of ordered lists in the summaries.

**Comment by gridhead** [comment]
@sopyb the pages for the meetings are generated on the fly with the use of templates available in the https://github.com/fedora-infra/mote/tree/main/mote/templates directory. The main.py file has the configuration for all the valid routes that the service allows for so those should lead you to the functions that generate the HTML files. The meeting logs and summaries are acquired from here https://meetbot-raw.fedoraproject.org/ so you might be able to experiment locally.

**Comment by raeeceip** [comment]
hi @gridhead is there a way to download them?

**Comment by sopyb** [comment]
> The meeting logs and summaries are acquired from here https://meetbot-raw.fedoraproject.org/ so you might be able to experiment locally.
Thanks for clarifying, @gridhead!

This is taking me way too long to wrap my head around. So, if I've got this right: the meeting logs and summaries are generated somewhere else. That means the fix I proposed earlier, where those would be modified on creation to display correctly in HTML, won't cut it.

If that's the case, then I'll move forward with adding the CSS style I mentioned before and make a pull request.

**Comment by sopyb** [comment]
> is there a way to download them?

@raeeceip Yes. All the logs and summaries are on https://meetbot-raw.fedoraproject.org

I used `wget -r -P "/srv/web/meetbot" -nH -R "index.html,robots.txt" -A ".html,.txt" https://meetbot-raw.fedoraproject.org/` to download all of them but if you want to download the logs of a specific channel you could just swqp `https://meetbot-raw.fedoraproject.org/` for `https://meetbot-raw.fedoraproject.org/fedora-meeting/`

**Comment by gridhead** [comment]
@sopyb could you please make changes to the README section where the developers are asked to download an archive from the Mega link and instead change it to provide the command that you mentioned in the comment https://github.com/fedora-infra/mote/issues/687#issuecomment-2002021203. Your command works great and does not have us rely on a third-party service to help our developers set up their environment.
